web/src/pages/Transaction/TransactionContent.tsx

```diff
@@ -0,0 +1,64 @@
+import {Tag, Typography} from 'antd';
+import {useNavigate} from 'react-router-dom';
+import TransactionHeader from '../../components/TransactionHeader';
+import {useTransaction} from '../../providers/TransactionRunDetail/Transaction.provider';
+import * as S from './Transaction.styled';
+
+const TransactionContent = () => {
+  const navigate = useNavigate();
+  const {transaction} = useTransaction();
+  return (
+    <>
+      <TransactionHeader onBack={() => navigate('/')} />
+      <S.Wrapper>
+        <S.Container>
+          <S.SectionLeft>
+            <Typography style={{fontSize: 16, marginBottom: 24}}>Execution steps</Typography>
+
+            {transaction?.steps.map(({name, version, ...test}, index) => {
+              return (
+                <S.Containerr data-cy={`run-card-${name}`} key={test.id}>
+                  <div>{index % 2 === 0 ? <S.IconFail /> : <S.IconSuccess />}</div>
```

Comment:
Can we have this done from the mock data instead?

Reply:
✅